docs/configuration/hyperparameter_optimization.md

```diff
@@ -230,6 +178,10 @@ executor:
   gpu_resources_per_trial: 1
   kubernetes_namespace: ray
   time_budget_s: 7200
+  scheduler:
+    type: async_hyperband
+    time_attr: training_iteration
+    reduction_factor: 4
```

Comment:
there's a weird indentation here

Reply:
I think it is correct indentation.  The keywords `time_attr` and `reduction_factor` are parameters for the `async_hyperband` scheduler: https://docs.ray.io/en/latest/tune/api_docs/schedulers.html#asha-tune-schedulers-ashascheduler

I'm leaving this unresolved for now.  If I'm missing something, I may need more help to understand the point.